src/main/java/com/gmail/goosius/siegewar/objects/Siege.java

```diff
@@ -312,27 +264,6 @@ public void addBannerControlSession(Player player, BannerControlSession bannerCo
 		bannerControlSessions.put(player, bannerControlSession);
 	}
 
-	/**
-	 * @return amount of time left as a String.
-	 */
-	public String getTimeRemaining() {
```

Comment:
This one probably also ought to be left in and deprecated. I could see it being used.